src/ig/IGExporter.ts

```diff
@@ -968,16 +966,16 @@ export class IGExporter {
     outputJSONSync(igJsonPath, this.ig, { spaces: 2 });
     this.updateOutputLog(
       igJsonPath,
-      [this.packagePath, path.join(this.igDataPath, 'ig.ini'), '{all input resources and pages}'],
+      [this.configPath, path.join(this.igDataPath, 'ig.ini'), '{all input resources and pages}'],
       'generated'
     );
     logger.info(`Generated ImplementationGuide-${this.ig.id}.json`);
   }
 
   /**
-   * Creates an ig.ini file based on the package.json and exports it to the IG folder.
-   * If the user specified an igi.ini file in the ig-data folder, then use its values
-   * as long as they don't conflict with values already in package.json.
+   * Creates an ig.ini file based on the "template" in config.yaml and exports it to the IG folder.
+   * If the user specified an igi.ini file in the ig-data folder, and no "template" in config.yaml is specified,
+   * the file from ig-data is used instead.
```

Comment:
Whoops. Guess I missed updating this comment, didn't I?

Reply:
Yeah you're right, `exampleConfig` could've been easier. But the old sunk cost got to me when I was halfway through changing things. And now this way we don't have unnecessary inconsistency across the tests.